src/rendermime/rendermime.ts

```diff
@@ -146,7 +146,14 @@ class RenderMime {
       if (model.data.has(mimeType)) {
         let options = { mimeType, model, sanitizer };
         let renderer = this._renderers[mimeType];
-        if (renderer.canRender(options)) {
+        let can_render = false;
+        try {
+          can_render = renderer.canRender(options);
+        } catch (err) {
+          console.error("Got an error when checking the renderer for the mimeType '"
```

Comment:
Let's use an interpolated string, which is a bit simpler:
```
`Got an error when checking the renderer for the mimeType ${mimeType}`
```
(I don't think we need the newline either)